### PR TITLE
adding a new servicekey.json for travis

### LINKED ISSUE #5305: regenerate google service account json
Try regenerating a new service account json for hellbender.  GenomicsDB looks for certain keys in the service account file which aren't present in the file we have in travis.

Lets try regenerating it to see if it's a field that's now included in the key file.

### PR DESCRIPTION
**Pull request by lbergelson**
* the old version didn't include certain keys in the json that are
necessary for GenomicsDB to authenticate
* closes #5305

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ before_install:
 #setup google cloud and github authentication
 - if [ $TRAVIS_SECURE_ENV_VARS == true ]; then
     export BOTO_CONFIG=/dev/null;
-    openssl aes-256-cbc -K $encrypted_29f3b7c4d8c3_key -iv $encrypted_29f3b7c4d8c3_iv  -in resources_for_CI/servicekey.json.enc -out servicekey.json -d;
+    openssl aes-256-cbc -K $encrypted_703d76169d63_key -iv $encrypted_703d76169d63_iv -in resources_for_CI/servicekey.json.enc -out servicekey.json -d
```
Comment (droazen) [inline .travis.yml]: Missing semicolon at end of line

**Comment by droazen** [comment]
@nalinigans Once this goes into master, could you try out your PR with this patch and see if the disabled failing tests pass now?

**Comment by droazen** [comment]
@lbergelson Looks like tests failed?

**Comment by whaleberg** [comment]
bleh, failures with 401 unauthorized.

**Comment by lbergelson** [comment]
I'm not sure what went wrong exactly, I might have messed up something when I did the travis encrypt command.   I've deleted the previous one, regenerated a new-new key, and reencrypted it.  Hopefully this time it will work.

**Pull request merged**